gtk4 4.20.0

danyeaw opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

danyeaw committed:
Update to GTK 4.20.0

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

danyeaw committed:
Fix gtk_doc option not found

danyeaw committed:
Fix librsvg not found

danyeaw committed:
Disable testsuite

danyeaw committed:
Fix minimum macOS SDK version

danyeaw commented:
The Windows build are failing due to not being able to find DirectX. I can build GTK4 on GitHub Actions runners, so I'm not sure why the WinSDK isn't sufficient for Azure Pipelines.

ryanvolz committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.24.17.59.41

Other tools:
- conda-build 25.7.0
- rattler-build 0.46.0
- rattler-build-conda-compat 1.4.6

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-recipe-manager`. The recipe can only be automatically migrated to the new v1 format if it is parseable by conda-recipe-manager.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18855431197](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18855431197). Examine the logs at this URL for more detail.</sub>

ryanvolz committed:
Add libxml2-devel to satisfy librsvg pkg-config

ryanvolz committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.27.15.27.40

Other tools:
- conda-build 25.7.0
- rattler-build 0.46.0
- rattler-build-conda-compat 1.4.6

ryanvolz committed:
Add libxml2214 migration

ryanvolz committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.27.15.27.40

Other tools:
- conda-build 25.7.0
- rattler-build 0.46.0
- rattler-build-conda-compat 1.4.6

ryanvolz committed:
Let meson fall back to subproject wrap for DirectX-Headers

ryanvolz committed:
Add patch to actually disable cups when we request it

conda-forge-admin commented:
Hi! This is the friendly conda-forge automerge bot!

I considered the following status checks when analyzing this PR:
 - **linter**: passed
 - **azure**: passed


Thus the PR was passing and merged! Have a great day!

danyeaw commented:
Thanks @ryanvolz for helping to get this in to a mergable state! I upstreamed the CUPS backend patch: https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/9089
